blog/se1/index.qmd

```diff
@@ -0,0 +1,28 @@
+---
+author: Caleb Kendra, Alish Chhetri, and Evan Nelson
+title: Programming Over Time
+date: '2023-11-29'
+date-format: YYYY
+categories: [post, developer development, software engineering]
```

Comment:
This should use the tag "professional development" and not "developer development". The SE articles are all connected to professional development.